src/App.js

```diff
@@ -1,17 +1,25 @@
+/* eslint-disable */
 import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Home from './components/Home';
+import Navbar from './components/Navbar';
+import Quote from './components/Quote';
 
-export default class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = '';
-  }
-
-  render() {
-    return (
-      <div className="App">
-        <Calculator />
+const App = () => (
+  <Router>
+    <div className="App">
+      <Navbar />
+      <div className="components">
+        <Routes>
+          <Route exact path="/" element={<Home />} />
+          <Route path="/calculator" element={<Calculator />} />
+          <Route path="/quote" element={<Quote />} />
```

Comment:
Well done! Your app consists of 3 pages: Home, Calculator, and Quote 🔥

Reply:
For every page, there should be a dedicated route to it made possible by [React Router](https://reactrouter.com/) 🙌